blockchains/ethereum/assets/0x946112efaB61C3636CBD52DE2E1392D7A75A6f01/info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Hydro Token",
```

Comment:
where to find official links to the contracts?

Reply:
[Here](https://projecthydro.medium.com/hydro-smart-contract-redeployment-57bf0e1f3fb4) is an official statement on the redeployment process

[Here](https://etherscan.io/address/0x946112efab61c3636cbd52de2e1392d7a75a6f01#code) is the official newly deployed contract

Do tell me if you need any more info

Reply:
Hi @vikmeup , do you need any more info about the new Hydro Token on eth?